spec/rails_helper.rb

```diff
@@ -48,7 +48,7 @@
 
 Capybara.register_driver :selenium_chrome_headless_sandboxless do |app|
   browser_options = ::Selenium::WebDriver::Chrome::Options.new
-  browser_options.args << '--headless'
```

Comment:
I think this shouldn't be uncommented? It's getting selenium errors in Travis...

Reply:
Dang, you're right. Fixing it now.